### PR TITLE
add Typora 0.9.89

### PR DESCRIPTION
**Pull request by ousugo**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/845)

### CODE REVIEW

File: manifests/Typora/Typora/0.9.87.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+LicenseUrl: http://support.typora.io/Acknowledgement/
+InstallerType: exe
+Installers:
+  - Arch: x64 # enumeration of supported architectures
+    Url: https://typora.io/windows/typora-setup-x64.exe
+    Sha256: ff5b0288c1c563bae4b1d4250d870e2bb758639b727815da45e658a5240a2cc4
+    Switches:
+      Silent: "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART"
+      SilentWithProgress: "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART"
+# ManifestVersion: 0.1.0
```
Comment (superusercode) [inline manifests/Typora/Typora/0.9.87.yaml:15]:
```suggestion
InstallerType: inno
Installers:
  - Arch: x64 # enumeration of supported architectures
    Url: https://typora.io/windows/typora-setup-x64.exe
    Sha256: ff5b0288c1c563bae4b1d4250d870e2bb758639b727815da45e658a5240a2cc4
# ManifestVersion: 0.1.0
```

Typora uses innosetup, so you can use the inno installertype and not need to include the switches.

**Review by superusercode — CHANGES_REQUESTED** [review]

File: manifests/Typora/Typora/0.9.87.yaml (hunk 1 above)
Comment (ousugo) [reply manifests/Typora/Typora/0.9.87.yaml:15]: thanks, I get it.

**Comment by superusercode** [comment]
odd installer scan errors

**Comment by superusercode** [comment]
@ousugo If you get the package smartscreen approved I recommend you use the fixed link (see how chocolatey does it at https://github.com/FranklinYu/chocolatey-packages/commit/f71dd2f957b9ccd8682cb03a86a150c4afb275b3#diff-7abe5604e6e47122488bf568b1780348) so that it won't break every time a new version comes out.

**Comment by superusercode** [comment]
I would update the current one with a fixed link but chocolatey never updated to that version and it will probably also fail smartscreen.

**Comment by ousugo** [comment]
Windows Defender confirmed the false positive.

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**